README.md

```diff
@@ -1,31 +1,35 @@
-#Project Name
-Hello Microverse Project
+#Project Name Hello Microverse!
+This project was build using Html and Css. I relied on git, github, and Vs Code Studio to complete this project.
 
-#Built With
-
-    Major languages: Html, css & Javascript
-    Technologies used: VS Code, Git & Github
 
+#Getting Started
+To get a local copy up and running follow these simple example steps.
+Prerequisites
+Setup
+Install
+Usage
+Run tests
+Deployment
+Authors
```

Comment:
- [ ] Just as the previous reviewers pointed, with patience copy the template [here](https://github.com/microverseinc/readme-template)  and paste it into your readme file. Then change the description, project name and all others to fit in for your project.